File: _Graph/vertex.py
class Vertex:
    def __init__(self, label: str, heuristic: int) -> None:
        self.__label: str = label
        self.__heuristic: int = heuristic
        
    def __str__(self) -> str:
        return self.__label + '(' + str(self.__heuristic) + ')'

File: _Graph/Vertex.java
package _Graph;
public record Vertex(String label, int heuristic) implements Comparable<Vertex> {
    @Override
    public int compareTo(Vertex other) {
        return Integer.compare(heuristic, other.heuristic);
    }
    
    @Override
    public String toString() {
        return label + "(" + heuristic + ")";
    }
}
